sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,181 @@
 import pytest
+import uuid
+import warnings
+import os
+
+from azure.data.tables import TableServiceClient
 from azure.eventhub.extensions.checkpointstoretable import TableCheckpointStore
+from azure.eventhub.exceptions import OwnershipLostError
+
+STORAGE_CONN_STR = [os.environ.get("AZURE_STORAGE_CONN_STR", "Azure Storage Connection String"), os.environ.get("AZURE_COSMOS_CONN_STR", "Azure Storage Connection String")]
+
+def get_live_storage_table_client(storage_connection_str):
+    try:
+        table_name = 'table{}'.format(uuid.uuid4().hex)
+        table_service_client = TableServiceClient.from_connection_string(storage_connection_str)
+        table_service_client.create_table_if_not_exists(table_name)
+        return storage_connection_str, table_name
+    except:
+        pytest.skip("Storage table client can't be created")
+
+def remove_live_storage_table_client(storage_connection_str, table_name):
+    try:
+        table_service_client = TableServiceClient.from_connection_string(storage_connection_str)
+        table_service_client.delete_table(table_name)
+    except:
+        warnings.warn(UserWarning("storage table teardown failed"))
+
+def _create_checkpoint(fully_qualified_namespace, eventhub_name, consumer_group,
+ partition_id, offset, sequence_number):
+    return {
+    'fully_qualified_namespace': fully_qualified_namespace,
+    'eventhub_name': eventhub_name,
+    'consumer_group': consumer_group,
+    'partition_id': str(partition_id),
+    'offset': offset,
+    'sequence_number': sequence_number
+            }
+
+def _create_ownership(fully_qualified_namespace, eventhub_name, consumer_group,
+ partition_id, owner_id, etag, last_modified_time):
+    return  {
+    'fully_qualified_namespace': fully_qualified_namespace,
+    'eventhub_name': eventhub_name,
+    'consumer_group': consumer_group,
+    'partition_id': str(partition_id),
+    'owner_id': owner_id,
+    'etag': etag,
+    'last_modified_time': last_modified_time
+            }
+
+def _claim_ownership_exception_test(storage_connection_str, table_name):
+    fully_qualified_namespace = 'test_namespace'
+    eventhub_name = 'eventhub'
+    consumer_group = '$default'
```

Comment:
make you can move these to the `_create_ownership` helper?

Reply:
so I need them for list_checkpoint and list ownership but I can add to the _create_ownership helper